public/CNAME

```diff
@@ -0,0 +1 @@
+hack.uclaacm.com 
```

Comment:
Please repush but exclude your /public folder changes (making sure your changes still look as expected beforehand)

If you look around, you may realize that the /public folder isn't included in the .gitignore-- this is because the blog post data is currently stored there so the deploy will properly link to it. I'll look further into whether or not we can .gitignore it later in the future.